WordPress/src/main/java/org/wordpress/android/models/CommentList.java

```diff
@@ -56,9 +61,10 @@ public boolean hasAnyWithStatus(CommentStatus status) {
      * returns true if any comments in this list do NOT have the passed status
      */
     public boolean hasAnyWithoutStatus(CommentStatus status) {
-        for (Comment comment: this) {
-            if (!comment.getStatusEnum().equals(status))
+        for (CommentModel comment : this) {
+            if (!status.toString().equals(comment.getStatus())) {
```

Comment:
It might be useful to add a `fromComment()` method to `CommentStatus`, just like [`PostStatus` has](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/9e535369eb7dfef24c844e90443b3c634d3c4bde/fluxc/src/main/java/org/wordpress/android/fluxc/model/post/PostStatus.java#L63). This would let us do an enum comparison here instead of String.

It will also let us replace instances of `CommentStatus commentStatus = CommentStatus.fromString(comment.getStatus());` with `CommentStatus commentStatus = CommentStatus.fromComment(comment);` (e.g. in `CommentAdapter.java`).